02.calendar/calender.rb

```diff
@@ -0,0 +1,31 @@
+require 'date'
+require 'optparse'
+
+params = ARGV.getopts("y:", "m:")
+y = params["y"]
+m = params["m"]
+
+def generate_target_date(y = nil, m = nil)
+  year = y.nil? ? Date.today.year : y.to_i
+  month = m.nil? ? Date.today.month : m.to_i
+  target_date = Date.new(year, month)
+end
+
+target_date = generate_target_date(y, m)
+
+puts target_date.strftime("%-m月 %Y").center(21)
+puts " 日 月 火 水 木 金 土"
```

Comment:
オリジナルのcalコマンドは

```suggestion
puts "日 月 火 水 木 金 土"
```

のように冒頭にスペースは出ないはずです。
そこも再現してほしいです〜。

Reply:
確認不足でした。修正いたしました。